tests/Bundle/Command/DebugResourceCommandTest.php

```diff
@@ -315,6 +317,7 @@ public function it_displays_the_metadata_for_given_resource_operation(): void
               routePrefix              null                      
               redirectToRoute          null                      
               redirectArguments        null                      
+              vars                     null                      
```

Comment:
Would be great to have those vars set up in one of these tests and check if they're properly displayed